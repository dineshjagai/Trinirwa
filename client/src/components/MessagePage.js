import React from 'react';
import './Home.css';
import NavBar from './navBar';
// import DisplayerTweets from './CenterDisplayerHome';
import DisplayerMessages from './CenterDisplayerMessages';
import { getCurrentReceiver } from '../auth/authServices';

export default function MessagePage() {
  const receiver = getCurrentReceiver();
  console.log(`receiver = ${receiver}`);

  return (
    <div>
      <NavBar />
      <div className="home">
        <div>
          <DisplayerMessages />
        </div>
      </div>
    </div>
  );
}
