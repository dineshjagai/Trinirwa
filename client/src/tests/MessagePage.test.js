import React from 'react';
import { createMount } from '@material-ui/core/test-utils';
import MessagePage from '../components/MessagePage';

test('Test MessagePage snapshot', () => {
  const component = createMount(<MessagePage />);
  expect(component).toMatchSnapshot();
});
