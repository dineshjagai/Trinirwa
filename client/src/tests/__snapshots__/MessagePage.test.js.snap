// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MessagePage snapshot 1`] = `[Function]`;
